main/src/vtr_lidar/src/modules/odometry/odometry_icp_module.cpp

```diff
@@ -498,13 +498,20 @@ void OdometryICPModule::run_(QueryCache &qdata0, OutputCache &,
     }
     *qdata.T_r_m_odo = T_r_m_eval->value();
     *qdata.timestamp_odo = query_stamp;
-#if 1
-    CLOG(WARNING, "lidar.odometry_icp") << "T_m_r is: " << qdata.T_r_m_odo->inverse().vec().transpose();
-    CLOG(WARNING, "lidar.odometry_icp") << "w_m_r_in_r is: " << qdata.w_m_r_in_r_odo->transpose();
-#endif
+//#if 1
+//    CLOG(WARNING, "lidar.odometry_icp") << "T_m_r is: " << qdata.T_r_m_odo->inverse().vec().transpose();
+//    CLOG(WARNING, "lidar.odometry_icp") << "w_m_r_in_r is: " << qdata.w_m_r_in_r_odo->transpose();
+//#endif
     //
     /// \todo double check validity when no vertex has been created
     *qdata.T_r_v_odo = T_r_m_icp * sliding_map_odo.T_vertex_this().inverse();
+
+    // Temporary modification by Jordy to test calibration of hte grizzly controller
```

Comment:
Is the modification temporary? Should this be removed?

Reply:
temporary modification, but we will need it for commissioning the new robot - I would leave as is for now and I can remove in my cleanup after you and Marija merge and I branch off of this.

Reply:
Marking as TODO